src/huntsman/pocs/utils/focus.py

```diff
@@ -109,15 +114,13 @@ def dark_image(self):
 
     @dark_image.setter
     def dark_image(self, image):
-        """ Set the dark image.
+        """ Set the dark image and update the dark mask.
         Args:
             image (np.array): The dark image array.
         """
-        if self._mask is None:
-            self._mask = np.zeros(shape=image.shape, dtype="bool")
-
         self._dark_image = image.astype(self._image_dtype)
-        self._mask = np.logical_or(self._mask, self._mask_saturated(self._dark_image))
+        self._dark_mask = self._mask_saturated(self._dark_image,
+                                               threshold=self._dark_mask_threshold)
```

Comment:
Nit picking at naming again, but I think something like "saturated_pixel_mask" would be better than "dark_mask" at conveying the meaning/purpose of this thing.